book/crud.rst

```diff
@@ -399,7 +399,7 @@ and register it as a service:
 .. code-block:: yaml
     :linenos:
 
-    # src/InventoryBundle/Resources/config/form.yml
+    # src/InventoryBundle/Resources/config/services.yml
```

Comment:
Good: this part of documentation help to Newcomer start & see result. services.yml, by default, are "autoloaded" = add to DependencyInjection when generate a bundle.

Bad: Newcomer will not be teached to do right style by our ideology: put form data in data.yml. 
How to:
Guess it will be right to left form.yml, but add a note, like "do not forget put data.yml to loader, like this: 
#OroCRM\Bundle\CaseBundle\DependencyInjection\OroCRMCaseExtension
...
 public function load(array $configs, ContainerBuilder $container)
...
$loader->load('form.yml');
"